src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -136,6 +173,22 @@ def load_arguments(self, _):
         c.argument('restore_to_staging_storage_account', arg_type=get_three_state_flag(), help='Use this flag when you want disks to be restored to the staging storage account using the --storage-account parameter. When not specified, disks will be restored to their original storage accounts. Default: false.')
         c.argument('target_resource_group', options_list=['--target-resource-group', '-t'], help='Use this to specify the target resource group in which the restored disks will be saved')
 
+    with self.argument_context('backup restore restore-azurefileshare') as c:
+        c.argument('resolve_conflict', arg_type=get_enum_type(['Overwrite', 'Skip']), options_list=['--resolve-conflict'], help="Instruction if there's a conflict with the restored data. Accepts Overwrite or Skip.")
+        c.argument('restore_mode', arg_type=get_enum_type(['OriginalLocation', 'AlternateLocation']), options_list=['--restore-mode'], help='Accepts OriginalLocation or AlternateLocation.')
```

Comment:
Please provide a definition or description for restore_mode argument and remove "Accepts OriginalLocation or AlternateLocation."